bcgov/HOUS-permit-portal
# Feature/bphh 613/review manager configure tip

## Description
- Adds ability for review managers to a make customizations to a published template and add a tip for a requirement block



## What type of PR is this? (check all applicable)

- [ X] 🍕 Feature

## Related Tickets & Documents
https://hous-bssb.atlassian.net/browse/BPHH-79


## Steps to QA
- Login as review manager
- Navigate to building permits page
- Click manage on any available permit template
- Click edit on any requirement block, and wait for the side bar to open
- Add a tip then click done
- Confirm that the tip is visible on the requirement block
- Click publish to save the changes

<img width="1728" alt="Screenshot 2024-03-05 at 11 13 49 AM" src="https://github.com/bcgov/HOUS-permit-portal/assets/32022335/f643c330-c221-4726-adb6-20c3ad30de6c">
